packages/wmr/src/lib/normalize-options.js

```diff
@@ -82,7 +94,11 @@ export async function normalizeOptions(options, mode) {
 
 			const fileUrl = url.pathToFileURL(configFile);
 			try {
-				custom = await eval('(x => import(x))')(fileUrl);
+				// Node <= 12.18.4 returns an empty module if we append search params to
```

Comment:
Idea for a workaround: we could create a temporary copy of the config and import that, then delete it (like we do with TS)? Though it doesn't seem worthwhile since this only affects server auto-restart after config changes.

Reply:
I think it's fair to require a somewhat recent node version. Only node versions `<=12.18.4` are affected, so everything starting from `12.19.0` will work. The newest version in the 12.x release line is `12.21.0`.